src/utils.js

```diff
@@ -210,24 +210,17 @@ export function datoComparator(a, b) {
     return a && b ? moment(a).diff(b) : (a ? 1 : 0) - (b ? 1 : 0);
 }
 
-function pad(number) {
-    return number < 10 ? `0${number}` : number;
-}
-
-export function toLocalDate(date) {
-    const dateObject = typeof date === 'string' ? new Date(date) : date;
-    return `${dateObject.getFullYear()}-${pad(dateObject.getMonth() + 1)}-${pad(
-        dateObject.getDate()
-    )}`;
-}
-
 export function HiddenIf({ hidden, children }) {
     if (hidden) {
         return null;
     }
     return children;
 }
 
-export function erTidspunktIPeriode(tidspunkt, fra, til) {
-    return moment(tidspunkt).isBetween(moment(fra), moment(til));
+export function getAntallDager(fraDato, tilDato) {
+    const msInDay = 1000 * 3600 * 24;
+    tilDato.setHours(0, 0, 0);
```

Comment:
Lurer på om dette blir feil
Tror Date i javascript bruker tidsonen du har satt på pc-en din. Så hvis du sitter i Norge, men satt at pc-en sin tidsone er i USA, så får vi helt feil dato antall dager her.. Tror vi har hatt dette problemet tidligere 🤔 og lurer på om vi gjorde noe i moment for å alltid sette timezonen til å være i Norge. 

Og tidszonen skal alltid være i forhold til norsk tid i aktivitetsplanen

Reply:
endret til bruk av moment